dev/NavigationView/NavigationView.cpp

```diff
@@ -83,6 +83,8 @@ static winrt::Size c_infSize{ std::numeric_limits<float>::infinity(), std::numer
 
 NavigationView::~NavigationView()
 {
+    // Used for correct displaying of initial display mode when DisplayMode is "Auto"
+    m_InitialNonForcedModeUpdate = false;
```

Comment:
Remove these two lines.
I guess what you want to do is in NavigationView::NavigationView

Reply:
Oh yes, you are right!